libpromises/evalfunction.c

```diff
@@ -8136,6 +8142,41 @@ void ModuleProtocol(EvalContext *ctx, char *command, const char *line, int print
         }
         break;
 
+    case '&':
+        // TODO: the variable name is limited to 256 to accommodate the
+        // context name once it's in the vartable.  Maybe this can be relaxed.
+        sscanf(line + 1, "%256[^=]=%4095[^\n]", name, content);
```

Comment:
So, as I understand it, a line will look like this:

```
&policy_varname=/path/to/data.json
```

Is that correct? Please add a comment here with an example like this. 

Also, `content` is not a good variable name. I suggest `file_name`.

Reply:
Then I need another variabele. The  variable `content` is used everywhere in the module protocol. So I just reuse this variable.  I will add an example